communities.json

```
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "nuclei-logo.png",
+      "title": "Nuclei",
+      "quote": "We connect with our community using our Discord server",
+      "inviteCode": "KECAGdH",
```

Do you want to use your new vanity invite code?
```suggestion
      "inviteCode": "projectdiscovery",
```